src/main/resources/templates/client/fragments/header.html

```diff
@@ -3,11 +3,15 @@
 <head>
 </head>
 <body>
-<div th:fragment="header(activeTab)">
+<div th:fragment="header(activeTab, backButtonPresent)">
```

Comment:
Misto backButtonPresent bych pridal parametr `parentScreen` nebo nejaky podobny, ve kterem se posleurl na stranku kam se ma pomoci back button uzivatel vratit. Jde o to, ze soucasna metoda `window.history.back()` vrati predposledni URL a tim padem muze vratit i posledni `post` request. Coz je spatne.. Vsude, kde se pak pouzije include toho menu se bude muset uvest jako parametr url toho rodicovskeho screenu.